src/module/ModuleBuilder.js

```diff
@@ -40,12 +40,14 @@ export class ModuleBuilder {
    * @param {String} moduleName
    * @param {Array} apiMethods
    * @param {Object} options additional objects for the module builder
+   * @param {String} presetModuleName
    */
-  constructor (store, api, moduleName, apiMethods, options = {}) {
+  constructor (store, api, moduleName, apiMethods, options = {}, presetModuleName = null) {
```

Comment:
While I am on board with the feature it's probably about time to extract and extend into a `PresetModuleBuilder`. The amount of arguments on the constructor is getting out of hand.

Reply:
I agree, but it seems out of the scope of the current fix, so I opened an issue: #87